media-sound/ncmpcpp/ncmpcpp-0.9.2_p20240724.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+NCMPCPP_COMMIT="68daf44032784a5b5b74781a9bf3826053eff4a2"
+
+DESCRIPTION="Featureful ncurses based MPD client inspired by ncmpc"
+HOMEPAGE="https://rybczak.net/ncmpcpp/"
+SRC_URI="https://github.com/ncmpcpp/ncmpcpp/archive/${NCMPCPP_COMMIT}.tar.gz -> ${P}.gh.tar.gz"
+S="${WORKDIR}/${PN}-${NCMPCPP_COMMIT}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="clock network outputs taglib visualizer"
+
+RDEPEND="
+	>=media-libs/libmpdclient-2.1
+	dev-libs/boost:=[icu,nls]
+	sys-libs/ncurses:=[unicode(+)]
+	sys-libs/readline:=
+	network? ( net-misc/curl )
```

Comment:
This doesn't seem to have any function in the ebuild? It's just pulling a dependency via RDEPEND which is against our policy.
https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001

as mentioned in the policy, use an `optfeature` for deps like this. Unless you simply forgot to add the configure option to this ebuild.

Reply:
I don't see any matching option when running `./configure --help` in the current master. I guess this means that this should be an optfeature.

Reply:
Nevermind, see my above comment regarding missing dependencies.

Reply:
Unfortunately, no. `net-misc/curl` must be where it was before. The binary links against libcurl, so it is needed both at build time (i.e. `DEPEND`) and run time (i.e. `RDEPEND`).

Furthermore, this cannot be an optional dependency. As far as I can tell, it is linked unconditionally. Thus, the new USE flag must be removed as well.

Reply:
You're right, the dependency is probably incorrectly labeled upstream.